src/components/cursor-controller.js

```diff
@@ -93,6 +93,8 @@ AFRAME.registerComponent("cursor-controller", {
     this.data.playerRig.addEventListener(this.data.primaryUp, this._handlePrimaryUp);
     this.data.playerRig.addEventListener(this.data.grabEvent, this._handlePrimaryDown);
     this.data.playerRig.addEventListener(this.data.releaseEvent, this._handlePrimaryUp);
+    this.data.playerRig.addEventListener("gamepadbuttondown", this._handlePrimaryDown);
```

Comment:
(apologies in advance, as I know this is a WIP PR.)

is this a correct branch? this appears to be for #364 perhaps?

btw, `gamepadbuttondown` are events that are proprietary to and work only in Firefox. (see w3c/gamepad#15 for context.)

Reply:
`gamepadbuttondown` is fired by https://github.com/donmccurdy/aframe-extras/blob/master/src/controls/gamepad-controls.js

Reply:
gd lol, I should've checked before spamming you with unnecessary info. thanks!